protocol/app/msgs/normal_msgs.go

```diff
@@ -267,19 +267,19 @@ var (
 		"/dydxprotocol.vault.MsgWithdrawFromMegavaultResponse": nil,
 	}
 
-	NormalMsgsConnect = map[string]sdk.Msg{
-		// connect marketmap messages
-		"/connect.marketmap.v2.MsgCreateMarkets":                   &marketmapmoduletypes.MsgCreateMarkets{},
-		"/connect.marketmap.v2.MsgCreateMarketsResponse":           nil,
-		"/connect.marketmap.v2.MsgParams":                          &marketmapmoduletypes.MsgParams{},
-		"/connect.marketmap.v2.MsgParamsResponse":                  nil,
-		"/connect.marketmap.v2.MsgRemoveMarketAuthorities":         &marketmapmoduletypes.MsgRemoveMarketAuthorities{},
-		"/connect.marketmap.v2.MsgRemoveMarketAuthoritiesResponse": nil,
-		"/connect.marketmap.v2.MsgRemoveMarkets":                   &marketmapmoduletypes.MsgRemoveMarkets{},
-		"/connect.marketmap.v2.MsgRemoveMarketsResponse":           nil,
-		"/connect.marketmap.v2.MsgUpdateMarkets":                   &marketmapmoduletypes.MsgUpdateMarkets{},
-		"/connect.marketmap.v2.MsgUpdateMarketsResponse":           nil,
-		"/connect.marketmap.v2.MsgUpsertMarkets":                   &marketmapmoduletypes.MsgUpsertMarkets{},
-		"/connect.marketmap.v2.MsgUpsertMarketsResponse":           nil,
+	NormalMsgsSlinky = map[string]sdk.Msg{
+		// slinky marketmap messages
+		"/slinky.marketmap.v1.MsgCreateMarkets":                   &marketmapmoduletypes.MsgCreateMarkets{},
+		"/slinky.marketmap.v1.MsgCreateMarketsResponse":           nil,
+		"/slinky.marketmap.v1.MsgParams":                          &marketmapmoduletypes.MsgParams{},
+		"/slinky.marketmap.v1.MsgParamsResponse":                  nil,
+		"/slinky.marketmap.v1.MsgRemoveMarketAuthorities":         &marketmapmoduletypes.MsgRemoveMarketAuthorities{},
+		"/slinky.marketmap.v1.MsgRemoveMarkets":                   &marketmapmoduletypes.MsgRemoveMarkets{},
+		"/slinky.marketmap.v1.MsgRemoveMarketsResponse":           nil,
+		"/slinky.marketmap.v1.MsgRemoveMarketAuthoritiesResponse": nil,
```

Comment:
nit: Ordering? RemoveMarketAuthoritiesResponse should come after RemoveMarketAuthorities?